src/app/core/config/config-fix.ts

```diff
@@ -673,7 +673,7 @@ export const defaultJsonConfig = {
                 "cols": [
                   ["health_bloodGroup"],
                   [
-                    { "id": "_description_health", "edit": "EditDescriptionOnly", "label": $localize`:description section:Health checkups are to be done regularly, at least every 6 months according to the program guidelines.`},
+                    { "id": "_description_health", "editComponent": "EditDescriptionOnly", "label": $localize`:description section:Health checkups are to be done regularly, at least every 6 months according to the program guidelines.`},
```

Comment:
There are still quite some places where "view" instead of "viewComponent" is used in this file.

Reply:
:+1:  will do the config-fix update now that we have settled on the format